addons/web_editor/static/src/js/wysiwyg/widgets/chatgpt_dialog.js

```diff
@@ -32,16 +32,11 @@ export class ChatGPTDialog extends Component {
         this._confirm();
     }
     formatContent(content) {
-        return markup([...this._postprocessGeneratedContent(content).childNodes].map(child => {
-            // Escape all text.
-            const nodes = new Set([...child.querySelectorAll('*')].flatMap(node => node.childNodes));
-            nodes.forEach(node => {
-                if (node.nodeType === Node.TEXT_NODE) {
-                    node.textContent = escape(node.textContent);
-                }
-            });
```

Comment:
Note: I must be missing something because I see no point of escaping text nodes as they are text by definition, when reading them through innerHTML/outerHTML they will be escaped naturally.